data-integration/admin_units.py

```diff
@@ -0,0 +1,120 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Created on Thu Oct 22 17:53:54 2020
+
+@author: giuseppeperonato
+"""
+
+import logging
+import os
+
+import geopandas as gpd
+import pandas as pd
+import sqlalchemy as sqla
+from pyproj import CRS
+
+import utilities
+
+# GISCO datasets GEOJSON EPSG:4326 1:1milion
+datasets = {
+    "countries": "https://gisco-services.ec.europa.eu/distribution/v2/countries/geojson/CNTR_RG_01M_2020_{}.geojson",
+    "nuts": "https://gisco-services.ec.europa.eu/distribution/v2/nuts/geojson/NUTS_RG_01M_2021_{}.geojson",
+    "lau": "https://gisco-services.ec.europa.eu/distribution/v2/lau/geojson/LAU_RG_01M_2019_{}.geojson",
+}
+
+# Dataset id
+DS_ID = 0
```

Comment:
I am not sure how a configuration file would help in this case. This script is only for this specific dataset, and the ID is not supposed to change. Do you have any suggestions?

Reply:
In the configuration file, you can put all the datasets IDs in the same place, and can easily check that you don't have twice the same dataset ID. You can also much more easily modify the dataset IDs in the case they change, even if they are not supposed to (maybe one day you have to remove dataset number X, or add a dataset between Y and Z). Going though 50 files to change  the IDs would be painful and prone to make mistakes!

Reply:
Ok, I see now. It makes sense indeed :-)

Reply:
:)

Reply:
What have you changed? I can't see it (the whole file is marked as changed)

Reply:
For this particular file, nothing. The DS_ID is still hardcoded because it is not a proper EnerMaps dataset, which we count starting from 1. For the real datasets, the DS_ID is read from the `datasets.csv` file, which makes it easier to update it.

Reply:
Ok, it was difficult to guess ;)